events/wc_dark_cult_events.txt

```diff
@@ -1086,7 +1147,378 @@ narrative_event = {
 				character_event = { id = MNM.10030 }
 			}
 			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5041 days = 7 }
+			character_event = { id = WCDAC.5045 years = 5 }
+		}
+	}
+}
+character_event = {
+	id = WCDAC.5041
+	desc = {
+		trigger = { NOT = { has_character_flag = hunted_by_garona } }
+		text = EVTDESC_WCDAC_5041_first_kill
+	}
+	desc = {
+		trigger = {
+			trigger = { has_character_flag = hunted_by_garona }
+		}
+		text = EVTDESC_WCDAC_5041_later_kills
+	}
+	picture = GFX_evt_ahnqiraj
+	border = GFX_event_normal_frame_intrigue
+
+	is_triggered_only = yes
+	
+	trigger = {
+		c_10620 = {
+			is_alive = yes
+			is_dying = no
+	
+			is_untouchable_trigger = no
+	
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+			is_inaccessible_trigger = no
+			
+			is_landed = no
+			
+			NOT = { religion = old_gods_worship }
+			NOT = { is_married = 52000 }
+			NOT = { obedient = 52000 }
+		}
+		character = 52000
+	}
+	
+	fail_trigger_effect = {
+		# Skip event
+		character_event = { id = WCDAC.5045 months = 12 }
```

Comment:
Are you sure it's needed at all? `WCDAC.5045` is fired in `WCDAC.5040`. Other firings of the event are very confusing.

Reply:
i kinda forgot about it after starting work on it again